nav2_bt_navigator/package.xml

```diff
@@ -13,13 +13,11 @@
   <build_depend>nav2_tasks</build_depend>
   <build_depend>nav2_msgs</build_depend>
   <build_depend>nav2_robot</build_depend>
-  <build_depend>behavior_tree_core</build_depend>
```

Comment:
How does rosdep know to download this library if we remove the dependency on it?

Reply:
> How does rosdep know to download this library...

How could it? rosdep knows nothing about this library until you add the keys to it. If we want to have a larger discussion about how rosdep works, it would be easier to jump on a call